packages/server-core-integration/README.md

```diff
@@ -72,7 +72,7 @@ ddp.connect((err: any) => {
 		return
 	}
 	let subId = ddp.subscribe('expectedMediaItems')
-	ddp.observe('expectedMediaItems', /* added cb */, /* changed cb */, /* removed cb */)
+	ddp.observe(PeripheralDevicePubSubCollectionsNames.expectedMediaItems, /* added cb */, /* changed cb */, /* removed cb */)
 
 	/* ... then later ... */
```

Comment:
While you're at it, could you add to the example an example of using the collection data as well?